src/path.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Paths used by Jenkins."""
+
+from pathlib import Path
+
+JENKINS_HOME = Path("/var/lib/jenkins")
```

Comment:
Can we move these to the charm.py file for cohesion?

Reply:
The [Jenkins charm](https://github.com/jenkinsci/jenkins-charm/blob/master/lib/charms/layer/jenkins/paths.py) has a lot of paths that will be defined later on in the future. Shall I move it to `charm.py` for now and create another module later on? I can foresee that it will need to be split up later on.
Thanks for the comment!

Reply:
I'd go with that and see how it evolves. It seems likely that some of the constants will only be used in charm.py, in which case they should be kept there

Reply:
Sounds good! Refactored :)